### PR TITLE
Editorial: convert headings to sentence case (#886)

### LINKED ISSUE #886: Use sentence case, not title case, for headings
WHATWG standards use sentence case for headings. Streams currently uses title case. So, currently Streams says:

> Transform Stream Default Controller Abstract Operations

This should instead be

> Transform stream default controller abstract operations

This does not affect too many headings, since a lot of headings are abstract operations or API surfaces, which are not affected by this.

### PR DESCRIPTION
**Pull request by AnthumChris**
Title Case Headings converted to Sentence case headings for h2,h3,h4,h5

closes #886

***
<a href="https://whatpr.org/streams/891.html" title="Last updated on Mar 6, 2018, 11:15 AM GMT (d90191c)">Preview</a> | <a href="https://whatpr.org/streams/891/bb99f6a...d90191c.html" title="Last updated on Mar 6, 2018, 11:15 AM GMT (d90191c)">Diff</a>

### CODE REVIEW

**Comment by ricea** [comment]
Great!

Just one thing I want to check with @domenic before landing: what do you want to do with "Queue-with-Sizes" in the "Queue-with-Sizes operations" heading?

File: index.bs [hunk 1]
```diff
@@ -4939,7 +4939,7 @@ CountQueuingStrategy({ <var>highWaterMark</var> })</h5>
   1. Return *1*.
 </emu-alg>
 
-<h3 id="queue-with-sizes">Queue-with-Sizes Operations</h3>
+<h3 id="queue-with-sizes">Queue-with-Sizes operations</h3>
```
Comment (domenic) [inline index.bs]: Yeah, let's make "sizes" lowercase

Comment (ricea) [reply index.bs]: @AnthumChris Can I ask you to do this?

Comment (AnthumChris) [reply index.bs]: @ricea All set, just pushed an `--amend` commit

**Comment by domenic** [comment]
Oh, do you mind adding yourself to the acknowledgments section?

**Comment by AnthumChris** [comment]
all set, thanks for including me

**Pull request merged**